src/content/developers/docs/nodes-and-clients/nodes-as-a-service

```diff
@@ -0,0 +1,95 @@
+---
+title: Nodes as a Service
+description: An entry-level overview of Node Services, the pros and cons, and popular providers.
+lang: en
+sidebar: true
+sidebarDepth: 2
+---
+
+## Introduction {#Introduction}
+
+Running your own [Ethereum node](/developers/docs/nodes-and-clients/#what-are-nodes-and-clients) can be challenging, especially when getting started or while scaling fast. There are a [number of services](/developers/docs/nodes-and-clients-nodes-as-a-service/#popular-node-services) that run optimized node infrastructures for you, so you can focus on developing your application or product instead. We'll explain how node services work, the pros and cons for using them and list providers if you are interested in getting started. 
+
+## Prerequisites {#prerequisites}
+If you don't already have an understanding of what nodes and clients are check out the [Nodes and Clients page](/developers/docs/nodes-and-clients/).
+
+## How do Node Services Work? {#how-do-node-services-work}
+
+Node services are providers run distributed node clients behind the scenes for you.
+
+These services typically provide an API key that you can use to write to and read from the blockchain, and often including access to [Ethereum testnets](/developers/docs/networks) in addition to the mainnet.
```

Comment:
Include trailing slash in link: `[Ethereum testnets](/developers/docs/networks/)`

Reply:
or even better link directly to testnets: `[Ethereum testnets](/developers/docs/networks/#testnets)` (anchors don't need trailing slashes)